### PR TITLE
Add kube-client

### PR DESCRIPTION
**Pull request by typeid**
Split up PR https://github.com/openshift/configuration-anomaly-detection/pull/279 into kube-client and the investigation

### CODE REVIEW

File: pkg/logging/logging.go [hunk 1]
```diff
@@ -29,8 +29,8 @@ func InitLogger(logLevelString string, clusterID string) *zap.SugaredLogger {
 	config.EncoderConfig.TimeKey = "timestamp"
 	config.Level = logLevel
 	config.EncoderConfig.EncodeTime = zapcore.RFC3339TimeEncoder
-	config.EncoderConfig.StacktraceKey = "" // to hide stacktrace info
-	config.EncoderConfig.CallerKey = ""     // to hide caller info
+	config.EncoderConfig.StacktraceKey = ""   // to hide stacktrace info
+	config.EncoderConfig.CallerKey = "caller" // to hide caller info
```
Comment (mrWinston) [inline pkg/logging/logging.go:33]: Question: Is the comment still correct with this change?

**Review by mrWinston — COMMENTED** [review]
Looks really good!

File: pkg/logging/logging.go (hunk 1 above)
Comment (typeid) [reply pkg/logging/logging.go]: Good catch, that's an artifact of the past :)

**Pull request merged**